src/components/camera/camera.ts

```diff
@@ -64,6 +76,27 @@ export interface Camera extends Resource {
    */
   getImage: (mimeType?: MimeType, extra?: Struct) => Promise<Uint8Array>;
 
+  /**
+   * Return a frame from a camera.
+   *
+   * @example
+   *
+   * ```ts
+   * const camera = new VIAM.CameraClient(machine, 'my_camera');
+   * const images = await camera.getImages();
+   * ```
+   *
+   * TODO(docs): include docs link for get images TS example
+   *
+   * @param filterSourceNames - A list of source names to filter the images by.
+   *   If empty or undefined, all images will be returned.
+   * @param extra - Extra parameters to pass to the camera.
+   */
+  getImages: (
+    filterSourceNames?: string[],
```

Comment:
Is filterSourceNames bound to a set of constant strings, such as `color`, `depth`, etc?

If so it would be helpful to make that a part of the type:

```suggestion
    filterSourceNames?: ('depth' | 'color')[],
```

Reply:
no it's not they can be arbitrary